### PR TITLE
os/bluestore: Cleanup

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-01-04_11:59:52-rados-wip-kefu-testing-2018-01-04-1836-distro-basic-smithi/

failures tracked by http://tracker.ceph.com/issues/22527 and http://tracker.ceph.com/issues/22596

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -11103,11 +11093,6 @@ int BlueStore::_set_alloc_hint(
   uint64_t expected_write_size,
   uint32_t flags)
 {
-  dout(15) << __func__ << " " << c->cid << " " << o->oid
-	   << " object_size " << expected_object_size
-	   << " write_size " << expected_write_size
-	   << " flags " << ceph_osd_alloc_hint_flag_string(flags)
-	   << dendl;
```
Comment (tchaikov) [inline src/os/bluestore/BlueStore.cc]: @liewegas is this change on purpose?

File: src/os/bluestore/BlueStore.cc [hunk 2]
```diff
@@ -11591,10 +11574,9 @@ int BlueStore::DBHistogram::get_key_slab(size_t sz)
 
 string BlueStore::DBHistogram::get_key_slab_to_range(int slab)
 {
-  int lower_bound = slab * KEY_SLAB;
-  int upper_bound = (slab + 1) * KEY_SLAB;
-  string ret = "[" + stringify(lower_bound) + "," + stringify(upper_bound) + ")";
-  return ret;
+  return "[" + stringify(slab * KEY_SLAB)     /* lower bound */
```
Comment (tchaikov) [inline src/os/bluestore/BlueStore.cc]: what's the rationale behind this change?

File: src/os/bluestore/BlueStore.cc (hunk 1 above)
Comment (liewegas) [reply src/os/bluestore/BlueStore.cc]: this should stay

**Comment by liewegas** [comment]
Looking at this again I'm not seeing any useful changes.. this is reshuffling code with other code that isn't any better, only different.
